src/lane_manager.hpp

```diff
@@ -12,7 +12,11 @@
 class LaneManager
 {
 public:
-		const float VillainSpawnProbability = 0.25; // We may want to make this level dependent in the future. Revise when levels are added.
+	struct tuple {
```

Comment:
maybe rename this to be more specific, given that the variables are pretty specific